docs/validator/staking.md

```diff
@@ -132,3 +132,18 @@ If you would like to see how much a validator is staking, you can run the comman
   storage_usage: 182
 }
 ```
+
+## Automatically re-staking
+
+This is implemented as a separate service that runs in parallel with `nearcore` and re-stakes if it sees that given validator might be kicked out or is already kicked out.
```

Comment:
you can run this service either on the same machine or on a different machine for extra security to keep signing key separate.

Reply:
✅